resources/service-catalog/charts/catalog-crds/values.yaml

```diff
@@ -115,3 +115,6 @@ servicePlanDefaultsEnabled: false
 ## by example :
 ## securityContext: { runAsUser: 1001 }
 securityContext: {}
+## image for migration process prevent SBU deletion
+migrationImagePath: eu.gcr.io/kyma-project/develop/service-catalog-upgrade
```

Comment:
this img has to general name. By reading that `service-catalog-upgrade` i think that is connected only with the service catalog which is not true :) please rename that